pages/create.js

```diff
@@ -0,0 +1,46 @@
+import NewOutivityForm from "@/components/NewOutivityForm";
+import { useRouter } from "next/router";
+import { useState } from "react";
+import { uid } from "uid";
+
+export default function CreateOutivity({ handleAddOutivity }) {
+  const router = useRouter();
+  const [selectedImage, setSelectedImage] = useState(null);
```

Comment:
what is the purpose of this state?

You appear to be using it in your `NewOutivityForm` component when you create a new outivity. 
selectedImage will only have a value in the moment of the form submission and will not be used afterwards.

I think you can safely remove this state and all its usages without having any affect on your application.

```suggestion
```

Reply:
Good point thanks for your advice

In our outivity form we had an image upload but now we`re going with a paste in url field for an image. We just forgot to remove all the code..

Reply:
Ah, I see, that explains it 👌